contracts/multisig-prover/src/encoding/evm.rs

```diff
@@ -55,6 +55,18 @@ impl TryFrom<Signer> for Operator {
     }
 }
 
+impl BatchID {
+    pub fn new(message_ids: &[String]) -> BatchID {
+        let mut message_ids = message_ids
+            .iter()
+            .map(|id| id.as_bytes())
+            .collect::<Vec<&[u8]>>();
+        message_ids.sort();
+
+        Keccak256::digest(message_ids.concat()).as_slice().into()
```

Comment:
Should join with a delimiter insatead of simply concat together.

Reply:
somehow I messed this up when I merged conflicts from previous PR and kept the old way of doing it